src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtCurrency.java

```diff
@@ -0,0 +1,27 @@
+package org.prebid.server.proto.openrtb.ext.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+import java.math.BigDecimal;
+import java.util.Map;
+
+/**
+ * Defines the contract for bidrequest.ext.prebid.targeting.currency
+ */
+@AllArgsConstructor(staticName = "of")
+@Value
+public class ExtCurrency {
```

Comment:
Right now this POJO looks exactly the same as `CurrencyConversionRates` while `bidrequest.ext.prebid.targeting.currency` field should have another schema - namely contain only one field with name "rates"